app/src/main/java/com/programmers/calculator/model/Expression.java

```diff
@@ -0,0 +1,29 @@
+package com.programmers.calculator.model;
+
+import com.programmers.calculator.util.Converter;
+import com.programmers.calculator.util.PostFixManager;
+
+public class Expression {
+    private final String expression;
```

Comment:
Expression를 객체로 뺀거 좋네요 👍
 보통 이런경우 상태를 value로 많이 사용합니다.

```suggestion
public class Expression {
    private final String value;
```